### PR TITLE
Updating dashboard

### PR DESCRIPTION
**Pull request by indiah444**
- Added coloured font to homepage.
- Added emojis to homepage.
- Deleted obsolete word cloud page.

### CODE REVIEW

File: dashboard/Home.py [hunk 1]
```diff
@@ -13,34 +13,34 @@
...
     ever-changing terrain by providing insights into how different media sources
-    from across the political spectrum cover key topics.
+    from across the political spectrum cover key topics. 
                 
     Through this dashboard, you can:
-    - Compare **topic coverage** between various news outlets.
-    - Analyse the **sentiment** of articles from different sources.
-    - Understand **how frequently** topics are covered and observe **shifts over time**.
-    - Visualise differences in **media framing**, empowering you to make more informed decisions about the information you trust.
+    - Compare <span style='color:blue; font-weight:bold;'>**topic coverage**</span> between various news outlets.
```
Comment (megansam5) [inline dashboard/Home.py:27]: ooo nice use of css

File: dashboard/Home.py [hunk 2]
```diff
@@ -13,34 +13,34 @@
 
     st.title("Welcome to the Media Polarisation Dashboard")
     st.markdown("""
-    ## Understanding Polarised Media Coverage and Sentiment
+    ## 📊 Understanding Polarised Media Coverage and Sentiment
```
Comment (megansam5) [inline dashboard/Home.py:16]: we love an emoji

**Review by megansam5 — APPROVED** [review]
A nice layout of the home page of the dashboard!
Oof the pylint score is not happy

**Comment by indiah444** [comment]
Fixed pylint score!

**Pull request merged**